src/guides/v2.3/architecture/archi_perspectives/components/modules/mod_and_areas.md

```diff
@@ -22,10 +22,12 @@ Magento is organized into these main areas:
 
 *  **Cron** (`crontab`): In `pub/cron.php`, the [`\Magento\Framework\App\Cron`]({{ site.mage2bloburl }}/{{ page.guide_version }}/lib/internal/Magento/Framework/App/Cron.php#L68-L70){:target="_blank"} class always loads the 'crontab' area.
 
-You can also send requests to Magento using the SOAP and REST APIs. These two areas
+You can also send requests to Magento using the SOAP, REST and GraphQL APIs. These two areas
```

Comment:
```suggestion
You can also send requests to Magento using the SOAP, REST and GraphQL APIs. These three areas
```

😉

Reply:
Hello @rogyar 
Oh, sorry. 
I have fixed it. Please check.
Thank you in advance.